.github/filters.yaml

```diff
@@ -1,28 +1,37 @@
+# This file determines when jobs should run or skipped based on which files were modified
+
 backend:
-  - '.github/actions/yarn-nm-install/*.yml'
-  - '.github/workflows/**'
-  - 'packages/**/package.json'
   - 'packages/**/server/**/*.(js|ts)'
   - 'packages/**/strapi-server.js'
   - 'packages/{utils,generators,cli,providers}/**'
   - 'packages/core/*/{lib,bin,ee,src}/**'
   - 'packages/core/database/**'
-  - 'yarn.lock'
-  - 'package.json'
+  - 'jest-preset.back.js'
 frontend:
-  - '.github/actions/yarn-nm-install/*.yml'
-  - '.github/workflows/**'
-  - 'packages/**/package.json'
   - 'packages/**/admin/src/**'
   - 'packages/**/admin/ee/admin/**'
   - 'packages/**/strapi-admin.js'
   - 'packages/admin-test-utils/**'
-  - 'yarn.lock'
-  - 'package.json'
+  - 'jest-preset.front.js'
+  - 'fileTransformer.js'
 api:
   - 'tests/api/**'
 e2e:
   - 'tests/e2e/**'
   - 'playwright.base.config.js'
 cli:
   - 'tests/cli/**'
+
+# TODO: this needs a lot more in it
+all_tests:
```

Comment:
should we name this global_sth ? so it makes it more explicit it's global files ? 🤔

Reply:
I originally didn't want it to affect docs_build or if we potentially added filters for lint/prettier, but thinking about it maybe it should, since package changes can affect pretty much anything anywhere.

I'll just call it global and have it trigger docs_build too